src/utilities/index.ts

```diff
@@ -1,29 +1,54 @@
 import Link from 'gatsby-link'
 
-export function checkBreakpoint(breakpoint) {
+export function checkBreakpoint(breakpoint: number) {
   if (typeof window !== `undefined`) {
     return window.matchMedia(`(min-width: ${breakpoint}px)`).matches
   }
 }
 
-export function externalUrl(url) {
+export function externalUrl(url: string) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [expected call-signature: 'externalUrl' to have a typedef](https://app.codacy.com/app/PrideInLondon/pride-london-web/pullRequest?prid=4553972)